src/jmxwrapper/jmx_wrapper.go

```diff
@@ -24,4 +24,10 @@ var (
 	// JMXClose is a wrapper around infra-integrations-sdk/jmx functions to allow
 	// easier mocking during tests
 	JMXClose = jmx.Close
+
+	// JMXOpen stores the value of the host where a JMX connection has been opened
+	JMXHost string
```

Comment:
it would be nice to refactor this in the future to have a wrapper struct with methods instead of these global variables . It looks a bit scary for me now how this variables are changed from the functions

Reply:
I totally agree with you, I don't love this approach, my initial thought was having a getter in the SDK and calling it from the wrapper. 

```
// connectionConfig is the configuration for the nrjmx command.
type connectionConfig struct {
	connectionURL         string
	hostname              string
	port                  string
	uriPath               string
	username              string
	password              string
	keyStore              string
	keyStorePassword      string
	trustStore            string
	trustStorePassword    string
	remote                bool
	remoteJBossStandalone bool
	executablePath        string
	verbose               bool
}

```
The getter would get the host and port from this struct in the SDK, but it would require a change in the SDK, do you think would be better to create a PR on the SDK with this getter methods?
Maybe I can Merge this as it is and open that PR and once accepted remove this and do it from there, what do you think?

Reply:
yes it would be better to add it to the sdk if not this is not super urgent to avoid the tech debt in my opinion

Reply:
Yes, you are right, I will freeze this make the change in the SDK and once it's present modify it in this PR

Reply:
cool , also cassandra and nrjmx integrations will get this easily in that way

Reply:
I have just added the new released SDK with the changes